package.json

```diff
@@ -1,7 +1,17 @@
 {
+  "name": "playwright",
+  "version": "1.0.0",
+  "type": "module",
+  "main": "index.js",
   "devDependencies": {
     "@playwright/test": "^1.45.2",
     "@types/node": "^20.14.11"
   },
-  "scripts": {}
+  "scripts": {
+    "test": "npx playwright test",
+    "report": "npx playwright show-report",
+    "test ui": "npx playwright test --ui",
+    "test stg": "npx playwright test --project=stg",
+    "test dev": "npx playwright test --project=dev"
```

Comment:
do not use spaces in commands name, use `:`

Reply:
I appreciate it